### PR TITLE
Feature/add env source for credentials (and rename which it's based on, which also has sep feature branch)

### PR DESCRIPTION
**Pull request by bglusman**
@tobowers I think this has you written all over it :-)  Remind me if we were going to add/change anything else besides the heroku env thing.

### CODE REVIEW

File: spec/lib/cluster-fsck/credential_grabber_spec.rb [hunk 1]
```diff
@@ -58,17 +65,34 @@ module ClusterFuck
...
 
-      it "should return the cluster-fuck credentials" do
+      it "should return the cluster-fsck credentials" do
         credential_grabber.find.should == cf_credentials
       end
     end
 
+    describe "when there is both ENV vars and a ~/.fog file" do
+      before :each do
+        ENV.stub(:[]).with("AWS_ACCESS_KEY_ID").and_return('env_access_key')
```
Comment (tobowers) [inline spec/lib/cluster-fsck/credential_grabber_spec.rb]: this should use the env_credentials['AWS_ACCESS_KEY_ID'] from the let above

File: lib/cluster-fsck.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    bucket_name
+  }
+
+  CLUSTER_FSCK_CONFIG = CLUSTER_FSCK_PATHS.map do |path_string|
+    path = File.expand_path(path_string)
+    YAML.load_file(path) if File.exists?(path)
+  end.compact.first || {}
+
+  CONFIG_BUCKET = CLUSTER_FSCK_CONFIG['BUCKET'] || SETUP_BUCKET.call
+  CLUSTER_FSCK_ENV = CLUSTER_FSCK_CONFIG['ENV'] || 'development'
```
Comment (tobowers) [inline lib/cluster-fsck.rb]: should be able to set this with an environment variable too (as one can now).  Should probably be able to skip all the files if you used an env variable... as in... the bucket should be settable there too.

**Comment by bglusman** [comment]
Just a ping that this is still waiting on review/merge, though getting everything over to use this version may be annoying at this point, dunno...

**Comment by bglusman** [comment]
Er, forget if that pings you without mentioning @tobowers sorry if you get twice :-)

**Pull request merged**